### DISCUSSION TITLE
href local file in sql

### DISCUSSION THREAD

**Discussion by MaximDown** [apache/superset#29305] [Q&A / Help]
Found an interesting [article ](https://superset-bi.ru/apache-superset-beautiful-table-with-link-and-image/)about embedding HTML into SQL

Need to create a link to a file on the local network, but SuperSet clears the href value

My sql ClickHouse dataset:
```
SELECT 
    NameCam, 
    `DateTime`, 
    NumCar, 
    CONCAT('file:\\\\test.company.ru\\FTP\\', NameCam, '\\', NameCam, '_', substring(replaceRegexpAll(CAST(`DateTime` AS String), '[-:. ]', ''), 1, 17), '.jpg') AS LinkImage 
FROM 
    Base.Table;
```

My column:
```
'<a href="'||"LinkImage"||'" target="_blank">'||"NumCar"||'</a>'
```

My result:
```
<a href="file:\\test.company.ru\FTP\0801 Perimetr Zabor\0801 Perimetr Zabor_20240613120834371.jpg" target="_blank">E345CC82</a>
```

Developer Console:
```
<a href="" target="_blank">A123BC45</a>
```

Accordingly, due to an empty link, the page reloads when clicked
It looks like SuperSet strips href, but I would like to use the file protocol

**Comment by jeverling** [comment]
I have the same question, is there a way to prevent Superset from stripping the href value, if we use e.g. a data URI?
The setting `HTML_SANITIZATION = False` doesn't seem to prevent the href value from being stripped here.
